shared/src/business/useCases/utilities/getCaseEntityForUpload.js

```diff
@@ -0,0 +1,47 @@
+const Case = require('../../entities/Case');
+const Message = require('../../entities/Message');
+const WorkItem = require('../../entities/WorkItem');
+const Document = require('../../entities/Document');
+const { DOCKET_SECTION } = require('../../entities/WorkQueue');
+
+exports.getCaseEntityForUpload = ({
+  user,
+  documentType,
+  caseToUpdate,
+  documentId,
+}) => {
+  const caseEntity = new Case(caseToUpdate);
+  const documentEntity = new Document({
+    userId: user.userId,
+    documentId,
+    filedBy: user.name,
+    documentType,
+  });
+
+  const workItem = new WorkItem({
+    sentBy: user.userId,
+    caseId: caseToUpdate.caseId,
+    document: {
+      documentId: documentEntity.documentId,
+      documentType,
+      createdAt: documentEntity.createdAt,
+    },
+    caseStatus: caseToUpdate.status,
+    assigneeId: null,
+    docketNumber: caseToUpdate.docketNumber,
+    section: DOCKET_SECTION,
+    assigneeName: null,
+  });
+  delete workItem.createdAt; // persistence layer won't save the workItem unless createdAt is null.... this is bad design
```

Comment:
We will try to address this comment in our current sprint.   This comment is related to the current approach we used for when you save a case with new work items in the document, it will only persist the work item when the createdAt date is not defined, but our entities will, by default, attach a createdAt date when you do `new WorkItem`